doc/contributor/release-branches.md

```diff
@@ -0,0 +1,49 @@
+# Releases Branches
+
+Binary suites are needed to run the GraalVM gate.
+Therefore, some care must be taken so they are deployed.
+
+### If there is no release branch in GraalVM
+
+Just make a PR against `master` as usual.
+The latest commit from TruffleRuby master will be picked up when creating the
```

Comment:
(just asking) when do you put a linebreak? For textfiles I usually put one whole sentence on one line.

Reply:
So it's pleasantly readable as a plain text file.